components/tier-page/index.js

```diff
@@ -162,7 +163,8 @@ class TierPage extends Component {
   render() {
     const { collective, tier, contributors, contributorsStats, redirect, LoggedInUser } = this.props;
     const canEdit = LoggedInUser && LoggedInUser.canEditCollective(collective);
-    const amountRaised = tier.interval ? tier.stats.totalRecurringDonations : tier.stats.totalDonated;
+    const amountRaised =
```

Comment:
This condition doesn't work, if `tier.interval` is `null` then `tier?.iterval !== INTERVALS.flexible` will return true